modules/rosa-hcp-sharing-vpc-dns-and-roles.adoc

```diff
@@ -0,0 +1,104 @@
+$ rosa create dns-domain --hosted-cp
+----
++
+The command creates a reserved `openshiftapps.com` DNS domain.
++
+[source,terminal]
+----
+I: DNS domain '14eo.p3.openshiftapps.com' has been created.
+I: To view all DNS domains, run 'rosa list dns-domains'
+----
+. Create an OIDC configuration.
++
+Review this article for more information on the link:https://access.redhat.com/articles/7031018[OIDC configuration process]. The following command produces the OIDC configuration ID that you need:
++
+[source,terminal]
+----
+$ rosa create oidc-config
+----
++
+You receive confirmation that the command created an OIDC configuration:
++
+[source,terminal]
+----
+I: To create Operator Roles for this OIDC Configuration, run the following command and remember to replace <user-defined> with a prefix of your choice:
+	rosa create operator-roles --prefix <user-defined> --oidc-config-id 25tu67hq45rto1am3slpf5lq6jargg
+----
+
+. Create the account roles by entering the following command:
++
+[source,terminal]
+----
+$ rosa create account-roles --oidc-config-id <oidc-config-ID> <1>
+    --installer-role-arn <Installer_Role> <2>
+    --route53-role-arn <Created_Route_53_Role_Arn> <3>
+    --vpc-endpoint-role-arn <Created_VPC_Endpoint_Role_Arn> <4>
+    --prefix <operator-prefix> <5>
+----
++
+--
+<1> Provide the OIDC configuration ID that you created in the previous step.
+<2> Provide your installer ARN that was created as part of the `rosa create account-roles` process.
+<3> Provide the ARN for the Route 53 role that the *VPC Owner* created.
+<4> Provide the ARN for the VPC endpoint role that the *VPC Owner* created.
+<5> Provide a prefix for the Operator roles.
+--
+
+. Create the Operator roles by entering the following command:
++
+[source,terminal]
+----
+$ rosa create operator-roles --oidc-config-id <oidc-config-ID> <1>
+    --installer-role-arn <Installer_Role> <2>
+    --route53-role-arn <Created_Route_53_Role_Arn> <3>
+    --vpc-endpoint-role-arn <Created_VPC_Endpoint_Role_Arn> <4>
+    --prefix <operator-prefix> <5>
+----
++
+--
+<1> Provide the OIDC configuration ID that you created in the previous step.
+<2> Provide your installer ARN that was created as part of the `rosa create account-roles` process.
+<3> Provide the ARN for the Route 53 role that the *VPC Owner* created.
+<4> Provide the ARN for the VPC endpoint role that the *VPC Owner* created.
+<5> Provide a prefix for the Operator roles.
+--
++
+[NOTE]
+====
+The Installer account role and the shared VPC roles must have a one-to-one relationship. If you want to create multiple shared VPC roles, you should create one set of account roles per shared VPC role.
+====
+
+. After you create the Operator roles, share the full domain name, which is created with `<intended_cluster_name>.<reserved_dns_domain>`, your _Ingress Operator Cloud Credentials_ role's ARN, your _Installer_ role's ARN, and your _Control plane Operator Cloud Credentials_ role's ARN with the *VPC Owner* to continue configuration.
```

Comment:
The domain doesn't necessarily need to be given to the VPC owner. In addition, `<intended_cluster_name>` is inaccurate, it should be `<domain-prefix>` and we should clarify that the domain prefix can be the cluster name or something else.

Reply:
Can we add an admonition that clarifies this?

Reply:
Hi @michaelryanmcneill - If they don't need to share the domain with the VPC owner, I'll just take that portion out. I then wouldn't need an admonition since that text isn't there. Correct?

Reply:
Not sure what would be the best way to highlight it but there are two models. The typical model would be having all network components in a separate account denominated network owner. But there is a second model where you can have the VPC in it but the hosted zones in the cluster owner account

Reply:
Would it make sense to write the docs for the typical model and then have a separate page in the [Tutorials section](https://docs.openshift.com/rosa/cloud_experts_tutorials/index.html) that outlines the second model? That would keep the two processes separate enough to not confuse users. However, I will most likely not get to the Tutorial until after the new year.

Reply:
I think that makes sense, would ask for @arendej or @michaelryanmcneill input on it